docs/README.md

```diff
@@ -36,31 +36,33 @@ is installed. `Succeeded` should appear in the **Status** column.
 1. Create the `openshift-operators` namespace (if it does not already exist).
 
     ```bash
-    $ kubectl create namespace openshift-operators
+    $ kubectl get ns openshift-operators || kubectl create ns openshift-operators
```

Comment:
Does it really make sense to use this long command instead of just creating the namespace? These docs are meant to be read and followed by users, so the shorter, the better. And if this command fails with "namespace already exists", the user will understand what's happening and will move on to the next step.

Reply:
I understand your POV, but sometimes users just copy/paste these commands into the console or create small scripts with them. Personally, I think it's a good idea to include a command that works in both scenarios. However, I don't have strong opinions on this and am happy to remove the change if you prefer. Do let me know.

Reply:
+1 to not add a long command that can hide things to the user. In the step description is already saying: create the namespace if does not already exist.

Reply:
@sridhargaddam Yeah, I see the benefits of both options. But I still think a shorter command is more useful for the majority of folks that will be reading this document. I assume that only a small number of readers will actually create scripts from this document, whereas a large majority will only read/scan through the document and possibly follow the steps manually; for those folks, shorter is better.

Reply:
Thanks guys. I'll revert this change. 

Do you want me to make changes at the following places as well?
1. https://github.com/istio-ecosystem/sail-operator/blob/f0d810ee1c462bde6d022f809b1d92bda2c99187/README.md?plain=1#L106
2. https://github.com/istio-ecosystem/sail-operator/blob/f0d810ee1c462bde6d022f809b1d92bda2c99187/README.md?plain=1#L114
3. https://github.com/istio-ecosystem/sail-operator/blob/f0d810ee1c462bde6d022f809b1d92bda2c99187/README.md?plain=1#L122
4. https://github.com/istio-ecosystem/sail-operator/blob/f0d810ee1c462bde6d022f809b1d92bda2c99187/docs/README.md?plain=1#L402

Reply:
Yes, I think is better to do the change on those places also and add the comment: "Create namespace xyz if it does not already exist"